src/transform/transform.ts

```diff
@@ -41,7 +41,7 @@ async function process(session: Session<CodeModel>) {
   // this must happen before processing objects as we depend on the
   // schema type being an actual Go type.
   for (const dictionary of values(session.model.schemas.dictionaries)) {
-    dictionary.elementType.language.go!.name = schemaTypeToGoType(session.model, dictionary.elementType, false);
+    dictionary.elementType.language.go!.name = schemaTypeToGoType(session.model, dictionary.elementType, true);
```

Comment:
Trying to understand what case we weren't covering before so that this switched to true....were we generating incorrectly for all in body params? Or are there cases where this should be false?

Reply:
The `inBody` param indicates if the type is passed via HTTP body or not (the "not" being a header).  When this is `true`, we add some extra annotations to the schema which is required for the intersection of additional properties and custom time times.
However, I think this parameter could possibly be removed as I don't think it matters if the extra annotations are there if we don't use them.

Reply:
Removing the param will erroneously add custom time types where they aren't needed, so it will remain for now.

Reply:
Ah got it, it's for additional props. Thanks!